.github/actions/binding_setup/action.yml

```diff
@@ -78,19 +78,31 @@ runs:
         r-version: "release"
         use-public-rspm: true
 
-    - name: "Configure ccache for R"
+    - name: Configure ccache for R
       if: inputs.lang == 'R'
       shell: bash
       run: |
-          ccache --set-config "sloppiness=include_file_ctime"
-          ccache --set-config "hash_dir=false"
-          ccache --zero-stats
```

Comment:
~We need those three lines to govern the `ccache` binary with respect to its behaviour for R files.~  Nevermind, found it below.

Also the comment in lin 81 is not quite correct for the `make` configuration (via `~/.R/Makevars`) that follows.

Reply:
Thanks, I tried to clarify the comment a bit (now it is `Configure R Makevars for ccache`).